pkg/agent/route/interfaces.go

```diff
@@ -80,4 +80,20 @@ type Interface interface {
 
 	// DeleteLocalAntreaFlexibleIPAMPodRule is used to delete related IP set entries when an AntreaFlexibleIPAM Pod is deleted.
 	DeleteLocalAntreaFlexibleIPAMPodRule(podAddresses []net.IP) error
+
+	// AddMulticlusterWireGuardRule is used to add IP rule for Multi-cluster WireGuard tunnel.
+	AddMulticlusterWireGuardRule(mark int, table int) error
+
+	// DeleteMulticlusterWireGuardRule is used to delete IP rule for Multi-cluster WireGuard tunnel when the Multi-cluster Gateway
+	// is deleted.
+	DeleteMulticlusterWireGuardRule(mark int, table int) error
+
+	// AddMulticlusterRoute is used to add static route on host for WireGuard tunnel.
+	AddMulticlusterRoute(index int, gw net.IP, table int) error
```

Comment:
ditto

Reply:
I'm not sure if we should rename this method, or we need to modify `AddRoutes` method, which is  focusing on PodCIDRs. And I saw there's another method named  `AddClusterIPRoute`, maybe we can keep it and use it  for multi-cluster rule only ?